### PR TITLE
Fix golint failures in pkg/registry/authentication/

### PR DESCRIPTION
**Pull request by CaoDonghui123**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:
Fix golint failures :
pkg/registry/admissionregistration/rest
pkg/registry/authentication/rest
pkg/registry/authentication/tokenreview

**Which issue(s) this PR fixes**:

Part of #68026

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: pkg/registry/admissionregistration/rest/storage_apiserver.go [hunk 1]
```diff
@@ -29,31 +29,33 @@ import (
 	validatingwebhookconfigurationstorage "k8s.io/kubernetes/pkg/registry/admissionregistration/validatingwebhookconfiguration/storage"
 )
 
-type RESTStorageProvider struct{}
+// StorageProvider is a REST storage provider for storage_apiserver
```
Comment (caesarxuchao) [inline pkg/registry/admissionregistration/rest/storage_apiserver.go:32]: `storage_apiserver` is not a user-facing term.

How about "StorageProvider is a REST storage provider for the admissionregistration API group."

Comment (caesarxuchao) [reply pkg/registry/admissionregistration/rest/storage_apiserver.go:32]: Also please add a "." at the end of each comment, see https://blog.golang.org/godoc.

File: pkg/registry/authentication/rest/storage_authentication.go [hunk 2]
```diff
@@ -29,12 +29,14 @@ import (
 	"k8s.io/kubernetes/pkg/registry/authentication/tokenreview"
 )
 
-type RESTStorageProvider struct {
+// StorageProvider is a REST storage provider for storage_authentication
```
Comment (caesarxuchao) [inline pkg/registry/authentication/rest/storage_authentication.go:32]: How about "StorageProvider is a REST storage provider for the authentication API group."

File: pkg/registry/admissionregistration/rest/storage_apiserver.go (hunk 1 above)
Comment (CaoDonghui123) [reply pkg/registry/admissionregistration/rest/storage_apiserver.go:32]: I get it , Thank  you so much

File: pkg/registry/authentication/tokenreview/storage.go [hunk 3]
```diff
@@ -34,26 +34,31 @@ import (
...
 	return false
 }
 
+// New creates a new tokenreview object.
 func (r *REST) New() runtime.Object {
 	return &authentication.TokenReview{}
 }
 
+// Create attempts to get self subject rules in specific namespace.
 func (r *REST) Create(ctx context.Context, obj runtime.Object, createValidation rest.ValidateObjectFunc, options *metav1.CreateOptions) (runtime.Object, error) {
```
Comment (caesarxuchao) [inline pkg/registry/authentication/tokenreview/storage.go:62]: Create reviews the token (the `obj`) and records the results in tokenReview.Status.

**Pull request closed without merging**